app/homepage/index.js

```diff
@@ -47,6 +47,22 @@ const addNumberOfEarthDays = () => {
 		.append(`${Math.round(earthDaysSinceLanding)} Earth days`);
 };
 
+const getPhotoOfTheDay = () => {
+	fetch('http://localhost:3000/apod')
+		.then((response) => {
```

Comment:
What happens if the response is a non-normal response (e.g. 500)?

Reply:
If the call to `/apod` fails, the Mars Rover image is rendered and the code to replace it fails gracefully. I would add a logger in here to capture the error.